docker/launch-netbox.sh

```diff
@@ -1,8 +1,33 @@
 #!/bin/bash
 
-UNIT_CONFIG="${UNIT_CONFIG-/etc/unit/nginx-unit.json}"
 UNIT_SOCKET="/opt/unit/unit.sock"
 
+put_config() {
+  RET=$(
+    curl \
+    --silent \
+    --write-out '%{http_code}' \
+    --request PUT \
+    --data-binary "@$1" \
+    --unix-socket $UNIT_SOCKET \
+    http://localhost/$2
+  )
+  RET_BODY=${RET::-3}
+  RET_STATUS=$(echo $RET | tail -c 4)
```

Comment:
```suggestion
  RET_BODY=${RET:0:-3}
  RET_STATUS=${RET: -3:3}     # Keep the space before the minus!
```

I believe an invocation of a subshell that runs `echo` and `tail` should not be necessary.